frontend/src/metabase/dashboard/components/DashboardHeader/DashboardHeader.styled.tsx

```diff
@@ -18,9 +18,17 @@ export const DashboardHeaderButton = styled(Button)<{
   visibleOnSmallScreen?: boolean;
   hasBackground?: boolean;
 }>`
-  padding: 0.25rem 0.5rem;
-  height: 2rem;
-  min-width: 2rem;
+  ${({ hasBackground }) =>
```

Comment:
Removes the padding for the button that's going to be used in the `ViewFooter`. There's definitely a better way to do this but I think for now this works well with both of the dashboard and question scenarios.